code/modules/clothing/glasses/glasses.dm

```diff
@@ -395,6 +395,12 @@
 	tint = 3				//to make them blind
 	prescription_upgradable = 0
 
+/obj/item/clothing/glasses/blindfold_fake
```

Comment:
Probably want to have the `prescription_upgradable` var here set to false. Like in normal blindfolds.

Reply:
Good idea